vendor/bat-native-ledger/src/bat/ledger/internal/state/state_migration_v9.cc

```diff
@@ -19,9 +19,14 @@ StateMigrationV9::StateMigrationV9(LedgerImpl* ledger) : ledger_(ledger) {}
 StateMigrationV9::~StateMigrationV9() = default;
 
 void StateMigrationV9::Migrate(ledger::ResultCallback callback) {
-  // Set the AC pref to false for all users located in a bitFlyer region.
-  if (ledger_->ledger_client()->GetBooleanOption(option::kIsBitflyerRegion))
+  auto client_info = ledger_->ledger_client()->GetClientInfo();
+
+  // Set the AC pref to false for all desktop users located in a bitFlyer
+  // region.
+  if (client_info && client_info->platform == mojom::Platform::DESKTOP &&
```

Comment:
Should we just use a `#if !defined(OS_ANDROID)` to handle this?

Reply:
Good point - fixed!